### PR TITLE
Add debian based corretto image

### PR DESCRIPTION
**Pull request by duncaan**
*Description of changes:*
Adds debian based corretto image.  Referenced https://github.com/docker-library/openjdk/blob/da44a9234d75e3d89ea00d679cb6a326cbbbbead/8/jdk/slim/Dockerfile for best practices.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ziyiluo** [comment]
@duncaan Thank you for your contribution. We are looking at it.

**Comment by duncaan** [comment]
@ziyiluo - were you all able to review this?

**Comment by ziyiluo** [comment]
@duncaan a quick follow up on this PR. We are drafting a guidance for the community-maintained Corretto docker images. We'd like to accept your contribution, but we are still discussing how the dockerfiles should be organized and managed. The debian-based dockerfile might be relocated to a different folder or even a different repository. Therefore, until all details are settled down, I prefer to put this PR on hold. Thank you for your patience!

**Comment by duncaan** [comment]
Thank you!

**Comment by duncaan** [comment]
@ziyiluo any update on this? I'll update this PR to the latest version if it's worthwhile.

**Comment by ziyiluo** [comment]
@duncaan thanks for waiting. I'm working on this right now. Do you mind I rebase your commit, move your dockerfile to a new directory (i.e., `contrib`), publish a new pull request and add you as a reviewer? I'll update your docker file to point to the latest Corretto version.

File: Dockerfile-debian
Comment (ziyiluo) [inline Dockerfile-debian]: @duncaan Corretto supports .deb installer (see [Corretto 8 release page](https://github.com/corretto/corretto-8/releases)). Would you modify the dockerfile to download and install .deb file instead (See [Corretto 8 installation instruction for Debian-based Linux](https://docs.aws.amazon.com/corretto/latest/corretto-8-ug/generic-linux-install.html))? (Or let me know if you prefer .tar.gz for a reason).

Comment (duncaan) [reply Dockerfile-debian]: I updated using the install instructions there.

**Comment by duncaan** [comment]
> @duncaan thanks for waiting. I'm working on this right now. Do you mind I rebase your commit, move your dockerfile to a new directory (i.e., `contrib`), publish a new pull request and add you as a reviewer? I'll update your docker file to point to the latest Corretto version.

I did this in the current PR - ready for your review.

**Review by ziyiluo — APPROVED** [review]
LGTM

**Comment by ziyiluo** [comment]
@duncaan I merged the PR but I'll move the Dockerfile and rename it tomorrow. Here's what I'll do:
1. Move `contrib/Dockerfile-debian` to `contrib/debian/Dockerfile` (I'll put your name as the committer since you contributed this).
2. Create `update-dockerfile.sh` that updates all Dockerfiles in this repo whenever a new Corretto version is available.

**Pull request merged**
